wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/environment/server/EnvironmentParser.java

```diff
@@ -228,15 +184,7 @@ private CheServicesEnvironmentImpl parseDocker(Environment environment) {
             }
         }
 
-        return composeEnvironment;
-    }
-
-    private String getContentOfRecipe(EnvironmentRecipe environmentRecipe) throws ServerException {
```

Comment:
Looks like this method doesn't depend on environment type, so you can leave it here